bindings/PKGS.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 
-PKGS="GLib GObject Atk Gio Secret cairo Pango PangoCairo GdkPixbuf Gdk-3.0 Gtk-3.0 Graphene Gdk-4.0 Gsk Gtk-4.0 GtkSource Soup GIRepository Poppler Vte Notify OSTree Ggit xlib GdkX11-3.0 GdkX11-4.0 Dbusmenu DbusmenuGtk3 Wnck Handy Gst GstBase GstAudio GstVideo GstTag GstPbutils JavaScriptCore-4.0 WebKit2 WebKit2WebExtension"
+PKGS="GLib GObject Atk Gio Secret cairo Pango PangoCairo GdkPixbuf Gdk-3.0 Gtk-3.0 Graphene Gdk-4.0 Gsk Gtk-4.0 GtkSource Soup GIRepository Poppler Vte Notify OSTree Ggit xlib GdkX11-3.0 GdkX11-4.0 Dbusmenu DbusmenuGtk3 Wnck Handy Gst GstBase GstAudio GstVideo GstTag GstPbutils JavaScriptCore-4.0 WebKit2 WebKit2WebExtension GtkSheet"
```

Comment:
These are the default packages to be built. Mostly as a convenience to myself, so if you take ownership of the package it would probably be best to remove `GtkSheet` from here, and re-add once it is easy to install in Fedora.